keyboards/QD60RGB/QD60_rgb.h

```diff
@@ -0,0 +1,83 @@
+ /**
```

Comment:
Please make the file and folder names lowercase, and remove the underscore from this .h and .c file.